mods/ra/maps/soviet-04b/main.lua

```diff
@@ -0,0 +1,151 @@
+CheckForRadar = function()
+	if not Radar.IsDead then
+		return true
+	end
+end
+
+CheckForCY = function()
+	if not CYard.IsDead then
+		return true
+	end
+end
+
+RunInitialActivities = function()
+	Harvester.FindResources()
+	Helper.Destroy()
+	IdlingUnits()
+	Trigger.AfterDelay(DateTime.Seconds(1), function()
+		BringPatrol1()
+		Trigger.AfterDelay(DateTime.Seconds(5), function()
+			BringPatrol2()
+		end)
+		BuildBase()
+	end)
+
+	Utils.Do(Map.NamedActors, function(actor)
+		if actor.Owner == Greece and actor.HasProperty("StartBuildingRepairs") then
+			Trigger.OnDamaged(actor, function(building)
+				if building.Owner == Greece and building.Health < 3/4 * building.MaxHealth then
+					building.StartBuildingRepairs()
+				end
+			end)
+		end
+	end)
+
+	Trigger.OnKilled(Powr, function(building)
+		BaseBuildings[1][4] = false
+	end)
+
+	Trigger.OnKilled(Barr, function(building)
+		BaseBuildings[2][4] = false
+	end)
+
+	Trigger.OnKilled(Proc, function(building)
+		BaseBuildings[3][4] = false
+	end)
+
+	Trigger.OnKilled(Weap, function(building)
+		BaseBuildings[4][4] = false
+	end)
+
+	Trigger.OnEnteredFootprint(VillageCamArea, function(actor, id)
+		if actor.Owner == player then
+			local camera = Actor.Create("camera", true, { Owner = player, Location = VillagePoint.Location })
+			Trigger.RemoveFootprintTrigger(id)
+			Trigger.OnAllKilled(Village, function()
+				camera.Destroy()
+			end)
+		end
+	end)
+
+	Trigger.OnAnyKilled(Civs, function()
+		Trigger.ClearAll(civ1)
+		Trigger.ClearAll(civ2)
+		Trigger.ClearAll(civ3)
+		local units = Reinforcements.Reinforce(Greece, Avengers, { SWRoadPoint.Location }, 0)
```

Comment:
`Fatal Lua Error: [string "main.lua"]:65: attempt to index global 'SWRoadPoint' (a nil value)`